assets/router/service-cloud.yaml

```diff
@@ -23,3 +23,4 @@ spec:
     protocol: TCP
     port: 443
     targetPort: https
+  sessionAffinity: None
```

Comment:
I'm curious to find out if any customers are currently setting `sessionAffinity` to a non-default value :)